### PR TITLE
[t2377] Tried adding drop-down with new, open, share, preview

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: src/editor/header.js [hunk 1]
```diff
@@ -2,12 +2,13 @@
  * If a copy of the MIT license was not distributed with this file, you can
  * obtain one at https://raw.github.com/mozilla/butter/master/LICENSE */
 
-define([], function(){
-  
+define([ "ui/widget/tooltip" ], function( Tooltip ){
```
Comment (secretrobotron) [inline src/editor/header.js]: Spacing before `{`.

File: src/editor/header.js [hunk 2]
```diff
@@ -40,6 +49,23 @@ define([], function(){
       }
     };
 
+    this.views = {
+      unSaved: function() {
+        _loginToShareTooltip.style.display = "";
```
Comment (secretrobotron) [inline src/editor/header.js]: Please use a class :).

File: css/header.less [hunk 3]
```diff
@@ -86,6 +86,7 @@
   max-width: 55%;
   min-width: 10%;
   height: 100%;
+  position: relative;
```
Comment (k88hudson) [inline css/header.less:89]: fixes the pencil icon position

File: src/editor/header.js [hunk 4]
```diff
@@ -17,6 +18,14 @@ define([], function(){
 
     var _currentFocus;
 
+    // Create a message for the disabled share editor.
+    _loginToShareTooltip = Tooltip.create({
+      title: "login-to-share",
+      message: "Login and save your project to share!",
```
Comment (humphd) [inline src/editor/header.js]: Drop the ! from this, capitalize 'Save' so it helps point them at the button:

```
Login and Save your project to share
```

File: src/editor/module.js [hunk 5]
```diff
@@ -47,6 +47,22 @@ define( [ "core/eventmanager", "core/trackevent", "./editor",
 
     _header = new Header( _editorAreaDOMRoot, _this );
 
+    function setupHeader() {
+      if ( butter.project && butter.project.isSaved ) {
```
Comment (humphd) [inline src/editor/module.js]: `butter.project` can't be null, this check is unnecessary.

**Pull request merged**